config/etfuturum/mixins.cfg

```diff
@@ -0,0 +1,161 @@
+# Configuration file
+
+##########################################################################################################
+# backported features
+#--------------------------------------------------------------------------------------------------------#
+# Backports that can typically only have a clean implementation with mixins.
+##########################################################################################################
+
+"backported features" {
+    # Adjusts the attenuation distance of certain sounds. This needs to be a separate mixin due to the way it works.
+    # Currently changes portal abience, and beacon ambience to have an attenuation distance of 8 blocks away, instead of 16.
+    # Modified Client Classes: net.minecraft.client.audio.SoundManager [default: true]
+    B:adjustedAttenuation=true
+
+    # Experimental: avoid dropping items when closing an inventory, like in modern versions.
+    # Modified Classes: net.minecraft.entity.player.EntityPlayerMP [default: false]
+    B:avoidDroppingItemsWhenClosing=false
+
+    # A port of Back in Slime, similar to how the elytra is a port of Backlytra. Allows pistons to interact with slime blocks. The original author of this is DonBruce64: https://legacy.curseforge.com/minecraft/mc-mods/back-in-slime-slime-blocks-for-1-7.
+    # Modified Classes: net.minecraft.block.BlockPistonBase [default: true]
+    B:betterPistons=true
+
+    # Allows some blocks without tile entities (e.g. composters) to interact with hoppers. May still not interact with modded pipes.
+    # Modified Classes: net.minecraft.tileentity.TileEntityHopper [default: true]
+    B:blockHopperInteraction=true
+
+    # Makes beds bouncy. Should work with most modded beds. For continuity disabling this also disables EFR beds being bouncy.
+    # Modified Classes: net.minecraft.block.BlockBed [default: true]
+    B:bouncyBeds=true
+
+    # In 1.14+, when breaking a block, the block break particles stay within the outline, instead of always occupying the whole block space.
+    # Mofified Client Classes: net.minecraft.client.particle.EffectRenderer [default: true]
+    B:boundedBlockBreakingParticles=true
+
+    # When greater than 1, boosts creative flight speed when sprinting, like in newer versions [range: 1.0 ~ 4.0, default: 2.0]
+    S:creativeFlightSpeedModifier=2.0
```

Comment:
have you tested that this really only affects creative mode and not the pseudo-creative of some jetpacks/chestplates?

Reply:
I have not. I'll set this to 1.0 for now and do some additional testing later if we decide we actually care about this config.